curator-framework/src/main/java/org/apache/curator/framework/imps/CreateBuilderImpl.java

```diff
@@ -83,6 +85,13 @@ public CreateBuilderMain orSetData()
         return this;
     }
 
+    @Override
+    public CreateBuilderMain withTtl(long ttl)
```

Comment:
I've started using strict camel case for everything. It's easier to remember. But, it's not a big deal to me.